packages/run-protocol/src/mintHolder.js

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+// @jessie-check
+
+import { makeIssuerKit } from '@agoric/ertp';
+
+/**
+ * This contract holds one mint; it basically wraps
+ * makeIssuerKit in its own contract, and hence in
+ * its own vat.
+ *
+ * @param {ContractFacet} zcf
+ * @returns {{ publicFacet: Issuer, creatorFacet: Mint }}
+ *
+ * @typedef {{
+ *   assetKind: AssetKind,
+ *   displayInfo: DisplayInfo,
```

Comment:
Should the type contain `keyword` too?

```suggestion
 *   displayInfo: DisplayInfo,
 *   keyword: string,
```